block_cascade/executors/databricks/resource.py

```diff
@@ -153,6 +156,16 @@ class DatabricksResource(BaseModel):
         remote task is run.
     timeout_seconds: int = 86400
         The maximum time this job can run for; default is 24 hours.
+    use_serverless: bool = False
+        If True, use Databricks serverless compute instead of provisioning a cluster.
+        When enabled, cluster-related parameters (worker_count, machine, spark_version,
+        cluster_policy, existing_cluster_id) are ignored.
+        See https://docs.databricks.com/api/workspace/jobs/submit for details.
+    serverless_environment_version: str = "3"
+        Serverless environment version. Each version comes with specific Python version
+        and set of preinstalled packages. Default is "3".
+        See https://docs.databricks.com/aws/release-notes/serverless/#serverless-environment-versions
+        Only used when use_serverless=True.
```

Comment:
consider bundling these related attributes together into an object.

Reply:
also consider adding any validation so conflicting arguments are not set by the user or its clear which takes precedence via logging.